libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/router/NavigationOffboardRoute.kt

```diff
@@ -29,7 +29,7 @@ import retrofit2.Callback
  * 1.0
  */
 
-class NavigationRoute
+class NavigationOffboardRoute
```

Comment:
This class can be internal, right?

Reply:
nope, this class is provided by developer

Reply:
Is it? As far as I can tell it's only used internally and the route options are the public component.